charts/newrelic-k8s-metrics-adapter/values.yaml

```diff
@@ -0,0 +1,89 @@
+# IMPORTANT: The Kubernetes cluster name
+# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+#
+# licenseKey:
+# cluster:
+# fargate:
+# IMPORTANT: the previous values can also be set as global so that they
+# can be shared by other newrelic product's charts.
+#
+# global:
+#   licenseKey:
+#   cluster:
+#   fargate
+
+personalAPIKey:
+
+accountID:
+externalMetrics:
+#  test1:
+#    query: "SELECT latest(cpuUsedCores) from K8sContainerSample "
+#    addClusterFilter: true
+
+# If not set, it will be automatically derived from global.licenseKey.
```

Comment:
does it work as well with the not global licenseKey?

Reply:
Maybe we can add a note like this
```suggestion
# If not set, it will be automatically derived from global.licenseKey.
# For US-based accounts, the region is: `US`.
# For EU-based accounts, the region is: `EU`.
# For Staging accounts, the region is: 'Staging' this is also automatically derived form `global.nrStaging`
```

Reply:
It should work fine with non global license key, as it make use of the common template:
```
{{/*
Return the licenseKey
*/}}
{{- define "newrelic-k8s-metrics-adapter.licenseKey" -}}
{{- if .Values.global}}
  {{- if .Values.global.licenseKey }}
      {{- .Values.global.licenseKey -}}
  {{- else -}}
      {{- .Values.licenseKey | default "" -}}
  {{- end -}}
{{- else -}}
    {{- .Values.licenseKey | default "" -}}
{{- end -}}
{{- end -}}
```